elastic/kibana
# [Monaco Editor] Add Search functionality

Fixes https://github.com/elastic/kibana/issues/186635

## Summary

This PR adds the find action to the Monaco code editor, which enables the search bar functionality as it is needed for Console with Monaco. The functionality is disabled by default.

<img width="1496" alt="Screenshot 2024-07-15 at 18 08 14" src="https://github.com/user-attachments/assets/a7476e4c-8ab8-4523-a815-1f7db483ddfa">

**How to test:**
1. Open Console and verify that Command/Ctrl + f opens the search bar (should work for both the editor and the output panel).
2. Open any other code editor in Kibana and verify that the search functionality is disabled.
